README.md

```diff
@@ -68,8 +68,7 @@ Finding projects depends on how the search API is structured for every hosting
 platform. For example, you can find all `publiccode.yml` on GitHub files by
 searching using the frontend or the API.
 
-* [GitHub Search `filename:publiccode.yml
-  path:/`](https://github.com/search?q=filename%3Apubliccode.yml+path%3A%2F)
+* [GitHub Search `path:/(^|\/)publiccode\.yml$/`](https://github.com/search?q=path%3A%2F%28%5E%7C%5C%2F%29publiccode%5C.yml%24%2F&type=code)
```

Comment:
Thanks @revolunet!

We can also go the simple route and do this, wdyt? cc @bvhme 

```suggestion
* [GitHub Search `path:publiccode.yml`](https://github.com/search?q=path%3Apubliccode.yml&type=code)
```

Reply:
indeed, it has even more results

Reply:
It having more results comes from it including files with other capitalisation (`Publiccode.yml`) and files where it is part of the path. Include more things but less aligned with the Standard. I like the more simple solution even though it is less useful in more precise or machine contexts.